indigo/analysis/refs/works.py

```diff
@@ -0,0 +1,184 @@
+import re
+
+from django.db.models import Q
+
+from cobalt import FrbrUri
+from docpipe.matchers import CitationMatcher
+from indigo.analysis.matchers import DocumentPatternMatcherMixin
+from indigo.plugins import plugins
+from indigo_api.models import Country, Work, WorkAlias
+from indigo_api.models.citations import CitationAlias
+
+
+class CommonTitlesCitationMatcher(DocumentPatternMatcherMixin, CitationMatcher):
+    """Finds references to works based "the XXX Act" patterns. """
+    # only supports English
+    locale = (None, 'eng', None)
+
+    html_candidate_xpath = ".//text()[contains(., ' the ') and contains(., ' Act') and not(ancestor::a)]"
+    xml_candidate_path = ".//text()[contains(., ' the ') and contains(., ' Act') and not(ancestor::ns:ref)]"
+
+    pattern_re = re.compile(r'\b(?P<pre>the\s+(?P<title>((?!the)\S+\s+){1,7})Act)\s*,?(\s*of\s+)?\s*(?P<year>[12][0-9]{3})?\b')
+
+    # map from titles to FRBR URIs
+    titles = None
+
+    def setup(self, frbr_uri, text=None, root=None):
+        self.setup_titles(frbr_uri)
+        super().setup(frbr_uri, text, root)
+
+    def setup_titles(self, frbr_uri):
+        self.titles = self.titles.get(frbr_uri.place) or self.titles.get(frbr_uri.country) or {}
```

Comment:
but where do they actually get set up lol 😆

Reply:
In `WorkTitlesCitationMatcher`